mne/io/ctf/tests/test_ctf.py

```diff
@@ -286,6 +286,7 @@ def test_saving_picked(tmpdir, comp_grade):
     temp_dir = str(tmpdir)
     out_fname = op.join(temp_dir, 'test_py_raw.fif')
     raw = read_raw_ctf(op.join(ctf_dir, ctf_fname_1_trial))
+    assert(raw.info['meas_date'] == (1367228160, 0))
```

Comment:
Did you also consider adding a check against MNE-C above in `test_read_ctf`? Ideally just adding `'meas_date'` to our list of keys here would work:

https://github.com/mne-tools/mne-python/blob/master/mne/io/ctf/tests/test_ctf.py#L118-L122

Though it's possible that it won't if MNE-C does not set the `meas_date` properly. In that case it's worth adding a comment up on those lines explicitly saying we don't check it because MNE-C does not set it the same way that we do.

Reply:
the mne-c fif files don't have `meas_date` set. so our fif reader guesses the `meas_date` so the test fails with :

```
E               AssertionError: meas_date
E               assert (991325820, 0) == (991340220, 0)
E                 At index 0 diff: 991325820 != 991340220
```

I added a comment saying that it should be revisited after mne-c addresses the issue.